app/controllers/portal/still_needs_helps_controller.rb

```diff
@@ -13,7 +13,7 @@ def update
 
       if current_client.update(update_params)
         if current_client.still_needs_help_yes?
-          current_client.tax_returns.where(status: "file_not_filing").each { |tax_return| tax_return.update!(status: "file_hold") }
+          current_client.tax_returns.where(state: "file_not_filing").each { |tax_return| tax_return.transition_to(:file_hold) }
```

Comment:
does this mean that some people could have gotten into a state where their tax return status was out of sync with their state machine transitions? not sure who was filling out this form and when (maybe no one!) in any case, if people's records did get out of sync, do we need to fix that somehow?

Reply:
i dont think we do because this feature wont affect any "active" tax returns that are around this season. i also dont think it would have that much of a reporting impact because we didn't update to use the state machine for tax return statuses until the tax season was closed.